File: python_kafka/consumer.py
from kafka import KafkaConsumer
consumer = KafkaConsumer(
    'numbers',                                  # Change topic name here!!!!
    bootstrap_servers=['192.168.252.100:9092', '192.168.252.100:9093', '192.168.252.100:9094'],
    group_id='names-consumer-group'
)

for message in consumer:
    print(message)

File: nodejs_kafka/Consumer.js
const { Kafka } = require('kafkajs')

const kafka = new Kafka({
  clientId: 'my-consumer',
  brokers: ['192.168.252.100:9092', '192.168.252.100:9093', '192.168.252.100:9094']  //Modify brokers here!!!!
})

const consumer = kafka.consumer({ groupId: 'consumer-group' })
const topic = 'numbers'                                             //Change topic here !!!!

const run = async () => {
  // Consuming
  await consumer.connect()

  await consumer.subscribe({ topic })

  await consumer.run({
    eachMessage: async ({ partition, message }) => {
      console.log({
        partition,
        offset: message.offset,
        value: message.value.toString(),
      })
    },
  })
}

run().catch(console.error)